appcenter-data/Data.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please remove **appcenter-data/Data.ts** and add **appcenter-data/Data.d.ts** as the type definition. See an example at https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-analytics/Analytics.d.ts.

Reply:
Done